src/common/tests/btree.c

```diff
@@ -830,8 +832,10 @@ ik_btr_drain(void **state)
 	int		 i;
 
 	D_ALLOC_ARRAY(arr, drain_keys);
-	if (arr == NULL)
+	if (arr == NULL) {
 		fail_msg("Array allocation failed");
+		return;
```

Comment:
fail_msg exits so this really should not be necessary.  It doesn't hurt anything, I suppose

Reply:
I agreed, just leave 'return' here to make coverity happy